stable/prometheus/values.yaml

```diff
@@ -1,8 +1,14 @@
+rbac:
+  create: false
+
 alertmanager:
   ## If false, alertmanager will not be installed
   ##
   enabled: true
 
+  # Defines the serviceAccountName to associate with a role when rbac is enabled.
+  serviceAccountName: default
```

Comment:
- [ ] You risk granting the "cluster-admin" _ClusterRole_ to the "default" service account here.